tests/test_dbapi.py

```diff
@@ -210,3 +210,19 @@ def test_get_badges_from_tags(api, dummy_issuer_id):
     assert len(badges_any) == 3
     badges_all = api.get_badges_from_tags(tags, match_all=True)
     assert len(badges_all) == 1
+
+
+def test_get_all_assertions(api, dummy_badge_id, dummy_person_id, dummy_issuer_id):
+    """Test get_all_assertions with default parameters."""
+    dummy_badge_bb = api.add_badge(
+        "TestBadge2", "TestImage2", "Another badge", "TestCriteria2", dummy_issuer_id
+    )
+
+    api.add_assertion(dummy_badge_id, "test@tester.com", None, "link1")
+    api.add_assertion(dummy_badge_bb, "test@tester.com", None, "link2")
+
+    assertions = list(api.get_all_assertions())
+
+    assert len(assertions) == 2
+    for i in range(len(assertions) - 1):
+        assert assertions[i].issued_on >= assertions[i + 1].issued_on
```

Comment:
It's a little bit surprising to use a loop here when you've asserted just above that there are only 2 assertions. This loop will only execute once, `i` will only be `0`, you can replace it with:

```suggestion
    assert assertions[0].issued_on >= assertions[1].issued_on
```